packages/web-api/scan-request/function.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bindings": [
+        {
+            "name": "scanRequest",
+            "type": "httpTrigger",
+            "direction": "in"
+        },
+        {
+            "type": "http",
+            "direction": "out",
+            "name": "res"
```

Comment:
can it be more meaningful?

Reply:
You mean name?

Reply:
yes name == res is this pre-set or custom?

Reply:
we can have any name here, I will change to name: scanResponse, this name will be available on context to assign response content.